### DISCUSSION TITLE
Exception Testing UsedDiskSpace Check

### DISCUSSION THREAD

**Discussion by jjalpha02** [spatie/laravel-health#131] [Q&A]
Hello community !!! i'm having an issue testing the UsedDiskSpace checks and I haven't found any question related to the render exception . The exception is : 

> Running check: Used Disk Space...
> Crashed}: `The check named `UsedDiskSpace` did not complete. An exception was thrown with this message: `Spatie\Regex\Exceptions\RegexFailed: Pattern `/(\d*)%/` with subject `` didn't capture a group named 1``

<img width="735" alt="Capture d’écran 2022-10-22 181440" src="https://user-images.githubusercontent.com/40037290/197356662-063acd69-efda-426c-8467-895c14a0bfee.png">

**Comment by GuilhermeLegramante** [comment]
I verified that the error occurs because the verification process is returning null, which causes the error to occur in the regex. I still don't know why the process isn't running.
